src/main/java/com/enonic/xp/app/welcome/market/MarketConfig.java

```diff
@@ -0,0 +1,8 @@
+package com.enonic.xp.app.welcome.market;
+
+public @interface MarketConfig
+{
+    String DEFAULT_GRAPHQL_URL = "https://market.enonic.com/api/graphql";
```

Comment:
This can be modified IF we ever have a need to customize domain separately from the url.
Currently this is an overkill.

Reply:
This is excessive, because domain can easily be fetched from the graphql url. 
Especially taking into account that we used `com.enonic.xp.market.cfg` config, not the app config.